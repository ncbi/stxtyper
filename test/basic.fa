>stxB2b_stop  gb|AAYVUU010000171.1| 
CTCAGCCGGACAGAGCAATTGCCTTCTAAGCAATCGGTCACTGGTTCGAATCCAGTACAACGCGCCATAC
TTATTTTTCTGGCTCGCTTTTGCGGGCCTTTTTTATATCTGCGCCGGGTCTGGTGCCGTATTACTTCAGC
CAAAAGGAACACCTGTATATGAAGTGTATATTGTTAAAATGGATACTGTGCCTGTTGCTGGGCTTTTCTT
CGGTATCCTATTCCCGGGAATTTACGATAGACTTTTCGACTCAACAAAGTTATGTCTCTTCGTTAAATAG
TATACGGACAGAAATATCGACTCCTCTTGAGCATATATCTCAGGGGACCACATCGGTGTCTGTTATTAAC
CATACCCCACCGGGCAGTTATTTTGCTGTGGATATACGAGGGCTTGATGTCTATCAGGCGCGTTTTGACC
ATCTTCGTCTGATTGTTGAGCAAAATAATTTATATGTGGCCGGATTCGTTAATACGGCAACAAATACTTT
CTACAGATTTTCTGATTTTGCACATATATCAGTGCCCGGTGTGACAACTGTTTCCATGACAACGGACAGC
AGTTATACCACTCTGCAACGTGTCGCAGCGCTGGAACGTTCCGGAATGCAAATCAGTCGTCACTCACTGG
TTTCATCATATCTGGCGTTAATGGAGTTTAGTGGAAATGCCATGACCAGAGATGCATCCAGAGCAGTTTT
GCGTTTTGTCACTGTCACAGCAGAAGCCTTACGGTTCAGGCAAATACAGAGAGAATTTCGTCTGGCACTG
TCTGAAACTGCTCCTGTTTATACGATGACACCGGAAGAAGTGGACCTCACACTGAACTGGGGGAGAATCA
GCAATGTGCTTCCGGAGTTTCGGGGAGAGGGTGGTGTCAGAGTGGGGCGAATATCCTTTAATAATATATC
AGCGATACTGGGCACAGTGGCGGTTATACTGAATTGCCATCATCAGGGGGCACGTTCCGTTCGCGCCGTG
AATGAAGAGATACAACCAGAATGTCAGATAACTGGCGACAGGCCAGTTATAAGGATAAACAATACTTTAT
GGGAAAGTAATACCGCAGCTGCTTTTCTGAATCGCAGGGCTCACTCTTTAAATACATCCGGAGAATAACA
GGAGTTAAATATGAAGAAGATATTTGTAGCGGCTTTATTTGCTTTTGTTTCTGTTAATGCAATGGCAGCT
GATTGTGCAAAAGGTAAAATTGAGTTCTCTAAGTATAATGAGAATGATACATTCACAGTAAAAGTGGCCG
GGAAAGAGTACTAGACTAACCGCTGGAATCTGCAACCGCTACTGCAAAGCGCACAGTTAACAGGAATGAC
GGTAACAATCAAATCAAATACCTGTGCGTCAGGTTCAGGATTTGCTGAAGTGCAGTTTAATTAATATCAG
AAGTATTGCTGGTTTCGTGGTGTGCAGCAATGTAGTTACAGTGCAATCAATGTCACAATTCAGTCAGTTG
ACAGTTGCCTGTCTGGCTGAGCATTTGTTAAAAAAATCCTGCATGATGAATCCCCCTGAGCGGAGGGGCA
TAATGACAGATGTTTGGTTGCGTATTGTATAGGCAAGTTGCGGATTCTGTCTGGTCATTGCAGAATTCAC
CGGGAGGCACCCGGCATCATGCTGTATACAGAGATTAGGCATATATCCAGGCTCCTCATCGCAGGAGCCT
TTTTACATGCAAAAAAAGCCCGCGCCGGGAGACGCGGGCAGCAAGGAATAAACAACAAAACGTGAAGTAA
TCAATTTTTCAGCTGGCGAATAATACCCGACAGTAATCACTCTGCGCAACTGCGCGGCCTTTTTCGTATT
GCGGGCTGTAGTCTTCCTCCAGTCATTGTCCTGTAACTTCCGGACTTCAGCCCGCCCCTTATCTGACTCA
CAACATTATCCCG

>stx2_fs gb|AAAFFU020000197.1| Escherichia coli strain 599478 SAMN10075572-rid9463843.guided.094, whole genome shotgun sequence
ATGCAATTCTGAGGGAATTACTGGATCAATCCACAGGAGTAATTATGACAAAACGTCGTAAGAAATACCA
GGAAAAAGAAGAGATTCGACACCCTGATTCACCTGAGGGATTAGTGGTAGCCGCAGCAAATAACAGGGCG
TTCGCAGAGCGCCTTGTTGGTGTTTACAGACTAGCCAAAGCAGGAGTGAAACATGGGCGTCGTTAAGTTA
GCTGATTACAGGCCTCAACTGGAGGTCGTGGAGCATCGCGTGGCAGATACCGAAGATGGTTTCATGCGCG
TTGCTAACGAGATTACCGACAGTCTGCTGATGGCTGATTTAACCGTCCGGCAGTTGAAGGTGATGCTCGC
TATCATGCGCAAGACATACGGATTCAATAAGCCGATGGATCGACTCACAAACACGCAGATAGCAGCCATG
ACAGGTATTCATCACACTCATGTTTGCGCTGCCAAGCGCCAGCTTATTGAGCGTAAATTCCTCATTGCTG
ATGGCGTGAAAATCGGAGTGAACAAGGTGGTTTCTCAGTGGATTAGCCAGGACAGCTTAACATTAGCTAA
AACAGCTAATAAAACATTAGCCAAGTCGGCTAATGGGTATAAGCCAAGTCAGCTAAACACAAAAGACAAT
ATACAAAAGACAATAAATACAAATACCCCCTTACCCCCTAACGGGGATGGCGATGGGCAGGTTAAACCTG
AACGTCGCAAGGCAGAACGAATCGACTACGAATCCTTCCTGAACGCCTACAACACCGAAGTCGGTGACAG
ACTTCCACACGCTGTTGCGGTCAACGAGAAACGAAAACGCCGCCTGAAGAAAATTATCCCGCAACTGAAA
ACGCCAAACGTGGACGGTTTCAGAGCGTATGTCAGGGCGTTTGTGCATCAGGCCAAGCCGTTTTACTTCG
GAGACAACGACACGGGCTGGACGGCAGATTTTGATTACCTGCTGAGGGAAGATTCGTTAACGGGAGTTCG
GGAAGGGAAGTTTGCAGACAGGGGGATTGCATGAGACAGGATATCGAAGCTAGCGTTATCGGTGGCCTGC
TGATTGGTGGATTAACTCCAACCGCCAGTGACGTTCTGGCAACGCTGGAGCCGGAAGCGTTTTCAATTCC
GCTCTACCGGAAAGCCTTCGAGGTTATTCGCAAGCAGGCGAGAAACAGAAACCTAATCGATGCGCTGATG
GTTGCCGAGGCGTGCGGAGAGGAGCATTTCACGTCAATCCTGATGACCAGCAAAAACTGCCCGAGTGCCG
CAAACCTGAAGGGATATGCCGGAATGGTCGCGGATAACTATCACCGCCGTCTGGTGCTGGAAATCATGGA
TGAAATGCGTGAACCAATCCAAAGCGGAACCATCGACGCATCGAGTCAGGCGATGGATGAACTTGTAAAG
CGTCTTTCAGCCATCAGAAAGCCCCGTGACGAGGTTAAACCTGTACGGTTAGGGGAAATCATCACTGACT
ACACTGACACGCTTGACAGGCGTCTGAGGAACGGAGAAGAGTCAGATACCCTGAAGACCGGAATCGAAGA
GCTTGACGCTATCACCGGAGGGATGAACGCAGAAGACCTGGTGATTATCGCCGCTCGTCCTGGTATGGGG
AAAACCGAACTGGCGCTGAAGATTGCCGAAGGTGTGGCAAGCCGCGTTATTCCTGGTTCTGACGTCCGGC
GCGGGGTATTGATTTTCTCAATGGAAATGAGCGCATTGCAGATTGCAGAGCGAAGCATTGCCAACGCCGG
GAGGATGTCGGTTAGCGTACTGCGAAATCCTGCATCGATGGATGACGAGGGCTGGGCACGTGTTGCTAAC
GGCATGAGTCAGCTTGCAGATTTGGATGTATGGGTAGTCGATGCTTCGCGGTTATCGGTCGAAGAAATAC
GCTCAATCGCAGAACGGCATAAGCAGGAAAATCCAAACCTGTCACTCATCATGGCGGATTATCTTGGCCT
GATTGAGAAGCCGAAAGCAGACCGCAACGACCTCGCAATTGCTCACATCTCCGGAAGCCTGAAGGCGATG
GCGAAAGACCTGAAAACACCGGTTATCTCCCTGAGTCAGCTTTCGCGCGATGTTGAGAAGCGACCAAACA
AACGCCCGACAAACGCAGATTTGCGTGATTCAGGAAGCATTGAACAGGACGCAGACTCAATCACACATCG
GTGTCTGTTATTAACCACACCCCACCGGGCAGTTATTTTGCTGTGGATATACGAGGGCTTGATGTCTATC
AGGCGCGTTTTGACCATCTTCGTCTGATTATTGAGCAAAATAATTTATATGTGGCCGGGTTCGTTAATAC
GGCAACAAATACTTTCTACCGTTTTTCAGATTTTACACATATATCAGTGCCCGGTGTGACAACGGTTTCC
ATGACAACGGACAGCAGTTATACCACTCTGCAACGTGTCGCAGCGCTGGAACGTTCCGGAATGCAAATCA
GTCGTCACTCACTGGTTTCATCATATCTGGCGTTAATGGAGTTCAGTGGTAATACAATGACCAGAGATGC
ATCCAGAGCAGTTCTGCGTTTTGTCAGTCACAGCAGAAGCCTTACGCTTCAGGCAGATACAGAGAGAATT
TCGTCAGGCACTGTCTGAAACTGCTCCTGTGTATACGATGACGCCGGGAGACGTGGACCTCACTCTGAAC
TGGGGGCGAATCAGCAATGTGCTTCCGGAGTATCGGGGAGAGGATGGTGTCAGAGTGGGGAGAATATCCT
TTAATAATATATCGGCGATACTGGGCACTGTGGCCGTTATACTGAATTGTCATCATCAGGGGGCGCGTTC
TGTTCGCGCCGTGAATGAAGATAGTCAACCAGAATGTCAGATAACTGGCGACAGGCCCGTTATAAAAATA
AACAATACATTATGGGAAAGTAATACAGCTGCAGCGTTTCTGAACAGAAAGTCACAGTTTTTATATACAA
CGGGTAAATAAAGGAGTTAAGTATGAAGAAGATGTTTATGGCGGTTTTATTTGCATTAGTTTCTGTTAAT
GCAATGGCGGCGGATTGCGCTAAAGGTAAAATTGAGTTTTCCAAGTATAATGAGAATGATACATTCACAG
TAAAAGTGGCCGGAAAAGAGTACTGGACCAGTCGCTGGAATCTGCAACCGTTACTGCAAAGTGCTCAGTT
GACAGGAATGACTGTCACAATCAAATCCAGTACCTGTGAATCAGGCTCCGGATTTGCTGAAGTGCAGTTT
AATAATGACTGAGGCATAACCTGATTCGTGGTATGTGGGTAACAAGTGTAATCTGTGTCACAATTCAGTC
AGTTGACAGTTGCCTGTCAGACTGAGCATTTGTTAAAAAAATTCGCATGGTGAATCCCCCTGTGCGGAGG
GGCGACTGGTGTATAAGGTGTCATCTCTGACGACAACCTGGATCGAGATACGCGGGTTCAGGGACACCGG
ACTGAACTCACCGGGAGGCACCCGGCACCATGTACATGATGATACAGATACGCAGTGTCAGCCCCTCTCC
GGAGGGGCTTTTTATCTGAATGATTCTGTTATTTTCGCCCGTATGGGCATATTTCCAGAATGCAGCGATG
ATTAAAACACTTATCTGTGATATTTCCTGTGTTTGCAGGGCACTCCTGGCTGTTTTTAATTAAATTCCAG
ACGTTTTTATTAAATGGGGCCACGTTGTAAATGGTGACGGGGAGCTCTTTATTATTGAGTATTACGCCTG
TCCGGGTTGGTGTAAAAATACAGCCCGGAGGGAGGAAGGTATCTGACTGATACCATAATATTAATTTTAT
GTTACACCATATGGCTGAAAATGATATGCCACATGCTGGCTGGATTACCGTGTCAATCACTATCCAGTTC
ATTTGCTTTCCTTATTTTCTTAAGGTTTTAAATGATGCCATAAATTATAGGTGATGTTTTGAAAGCAATC
TGTCAGGCTGAGTTGTTTTTCTGGTGTCGTTTATTCTTTGTTTAAAATAAGAGGGCATTTTAAATTGTCG
GCGTTTTTTTGTTATATAAGAGACATGGTTTTTTTATAGCGTGTTTTTTATGTGTTACATGATTGTGTGT
TCCGGATAATTACTTTGCTGGCGGCAGGAGAGGTTATGACATTTAAACACTACGATGTGGTCAGGGCGGC
ATCGCCGTCAGACCTTGCTGAACGACTGACACAAAAACTGA


>stx2_novel gb|AARSJU010000054.1| Escherichia coli strain PNUSAE016060 SAMN09787738-rid9281453.guided.46, whole genome shotgun sequence
GGCTGCGCGGGGTTCGAGTCCTCGATGGCGGTCCATTATCGGTATTCAGCGTTGTTAGCTCAGCCGGACA
GAGCAATTGCCTTCTAAGCAATCGGTCACTGGTTCGAATCCAGTACAACGCGCCATATTTATTTACCAGG
CTCGCTTTTGCGGGCCTTTTTTATATCTGCGCCGGGTCTGGTGCTGATTACTTCAGCCAAAAGGAACACC
TGTATATGAAGTGTATATTATTTAAATGGGTACTGTGCCTGTTACTGGGTTTTTCTTCGGTATCCTATTC
CCGGGAATTTACGATAGACTTTTCGACTCAACAAAGTTATGTCTCTTCGTTAAATAGTATACGGACAGAG
ATATCGACCCCTCTTGAACATATATCTCAGGGGACCACATCGGTGTCTGTTATTAACCACACCCCGCCGG
GCAGTTATTTTGCTGTGGATATACGAGGGCTTGATGTCTATCAGGCGCGTTTTGACCATCTTCGTCTGAT
TATTGAGCAAAATAATTTATATGTGGCCGGGTTCGTTAATACGGCAACAAATACTTTCTACCGTTTTTCA
GATTTTACACATATATCAGTGCCCGGTGTGACAACGGTTTCCATGACAACGGACAGCAGTTATACCACTC
TGCAACGTGTCGCAGCGCTGGAACGTTCCGGAATGCAAATCAGTCGTCACTCACTGGTTTCATCATATCT
GGCGTTAATGGAGTTCAGTGGTAATACAATGACCAGAGATGCATCCAGAGCAGTTCTGCGTTTTGTCACT
GTCACAGCAGAAGCCTTACGCTTCAGGCAGATACAGAGAGAATTTCGTCAGGCACTGTCTGAAACTGCTC
CTGTGTATACGATGACGCCGGGAGACGTGGACCTCACTCTGAACTGGGGGCGAATCAGCAATGTGCTTCC
GGAGTATCGGGGAGAGGATGGTGTCAGAGTGGGGAGAATATCCTTTAATAATATATCGGCGATACTGGGT
ACTGTGGCCGTTATACTGAATTGCCATCATCAGGGGGCGCGTTCTGTTCGCGCCGTGAATGAAGAGAGTC
AACCAGAATGTCAGATAACTGGCGACAGGCCCGTTATAAAAATAAACAATACATTATGGGAAAGTAATAC
AGCAGCAGCGTTTCTGAACAGAAAATCACAGTCTTTATATACAACGGGTAAATAAAGGAGTTAAGTATGA
AGAAGATGTTTATGGCGGTTTTATTTGCATTAGTTTCTGTTAATGCAATGGCGGCGGATTGCGCTAAAGG
TAAAATTGAGTTTTCCAAGTATAATGAGAATGATACATTCACAGTAAAAGTGGCCGGGAAAGAATACTGG
ACCAGTCGCTGGAATCTGCAACCGTTACTGCAAAGTGCTCAGCTGACAGGAATGACTGTCACAATCAAAT
CCAGTACCTGTGAATCAGGCTCCGGATTTGCTGAAGTGCAGTTTAATAATGACTGAGGCATAACCTGATT
CGTGGTATGTGGGTAACAAGTGTAATCTGTGTCACAATTCAGTCAGTTGACAGTTGCCTGTCAGACTGAG
CATTTGTTAAAAAA

>stx1a gb|AAXXHU010000093.1| Escherichia coli strain PNUSAE065727 SAMN17086846-rid11246123.denovo.095, whole genome shotgun sequence
CATCCTGTGTATCAATAAATGTTGTTATCTACGTACGTCAAGTAGTCGCATGAGATCTGACCAGATATGT
TAAGGTTGCAGCTCTCTTTGAATATGATTATCATTTTCATTACGTTATTGTTACGTTTATCCGGTGCGCC
GTAAAACGCCGTCCTTCAGGGCGTGGAGGATGTCAAGAATATAGTTATCGTATGGTGCTCAAGGAGTATT
GTGTAATATGAAAATAATTATTTTTAGAGTGCTAACTTTTTTCTTTGTTATCTTTTCAGTTAATGTGGTG
GCGAAGGAATTTACCTTAGACTTCTCGACTGCAAAGACGTATGTAGATTCGCTGAATGTCATTCGCTCTG
CAATAGGTACTCCATTACAGACTATTTCATCAGGAGGTACGTCTTTACTGATGATTGATAGTGGCACAGG
GGATAATTTGTTTGCAGTTGATGTCAGAGGGATAGATCCAGAGGAAGGGCGGTTTAATAATCTACGGCTT
ATTGTTGAACGAAATAATTTATATGTGACAGGATTTGTTAACAGGACAAATAATGTTTTTTATCGCTTTG
CTGATTTTTCACATGTTACCTTTCCAGGTACAACAGCGGTTACATTGTCTGGTGACAGTAGCTATACCAC
GTTACAGCGTGTTGCAGGGATCAGTCGTACGGGGATGCAGATAAATCGCCATTCGTTGACTACTTCTTAT
CTGGATTTAATGTCGCATAGTGGAACCTCACTGACGCAGTCTGTGGCAAGAGCGATGTTACGGTTTGTTA
CTGTGACAGCTGAAGCTTTACGTTTTCGGCAAATACAGAGGGGATTTCGTACAACACTGGATGATCTCAG
TGGGCGTTCTTATGTAATGACTGCTGAAGATGTTGATCTTACATTGAACTGGGGAAGGTTGAGTAGCGTC
CTGCCTGACTATCATGGACAAGACTCTGTTCGTGTAGGAAGAATTTCTTTTGGAAGCATTAATGCAATTC
TGGGAAGCGTGGCATTAATACTGAATTGTCATCATCATGCATCGCGAGTTGCCAGAATGGCATCTGATGA
GTTTCCTTCTATGTGTCCGGCAGATGGAAGAGTCCGTGGGATTACGCACAATAAAATATTGTGGGATTCA
TCCACTCTGGGGGCAATTCTGATGCGCAGAACTATTAGCAGTTGAGGGGGTAAAATGAAAAAAACATTAT
TAATAGCTGCATCGCTTTCATTTTTTTCAGCAAGTGCGCTGGCGACGCCTGATTGTGTAACTGGAAAGGT
GGAGTATACAAAATATAATGATGACGATACCTTTACAGTTAAAGTGGGTGATAAAGAATTATTTACCAAC
AGATGGAATCTTCAGTCTCTTCTTCTCAGTGCGCAAATTACGGGGATGACTGTAACCATTAAAACTAATG
CCTGTCATAATGGAGGGGGATTCAGCGAAGTTATTTTTCGTTGACTCAGAATAGCTCAGTGAAAATAGCA
GGCGGAGATTCATAAATGTTAAATACATCTCAATTCAGTCAGTTGTTGCCGGTCTGATAATAGATGTGTT
AGAAAATTTCTGCATGGTGAATCCCCCTGTGCGGAGGGGCGACTGGTGAACGGTATGATCTCTTTGATGA
TCGTAAGCGAGAATACGCGGGTTTGGTGGCACCAGGCCGAACTCACCGGGAGGCACCCGGCATCATGCTG
TATACAGAGATTAGGCATATATCCAGGCTCCTCATCGCAGGAGCCTTTTTACATGCAAAAAAAAGCCCGA
GTGGGTTCGGGCAACAGCATGAGATACTTGCATTGTCATTTTTATCGTGCGGATTTTAACCAGGATTCAT
AAGGCTGCGCAACTGCGCGGCCTTTTTCGTATTTCGGGCTGTAGTCCCCGTGTGTCATTCAGGCTTCCGG
ACTACAGCCCACTCCATATCTGATTTAATACACTATCCCGGCCGGGAGGAATAATGACA

>stx2c gb|AAASXN010000098.1| Listeria monocytogenes strain 104657 SAMN11191667-rid7376543.denovo.100, whole genome shotgun sequence
CGGCTCATGTCCTGCACATCATGCAGACAATGGTCCGCCGGAATGATGTCGTTATATCTGCATGCTGCAC
CACCCGGTGTCACCGTACTGCGACGCGCCAGTTGCTTAATACGCGGCTCCGGACGGTCATATGTCTCCGG
CAGCGGAAGACCTTCACCATACGACATGCCGTTTGACTGCCCTGCCAGAACCACAACAAAGTAATACTCC
GGGGCTCTGGTGGCACTGACAGCCACCGCCTTACCCTCATTACCGGTCACCGCCACTGGTGTGGTGACAT
CACCTTCCGCCGCAATGGCCTGCATCAGGGTGTAAGGCGTGATGGCCACCGGACTGCCAAATGGCTGCCA
GCCCTCCTTCAGTTTTTGTGTCAGTCGTTCAGCAAGGTCTGACGGCGATGCCGCCCTGACCACATCGTAG
TGTTTAAATGTCATAACCTCTCCTGCCGCCAGCAAAGTAATTATCCGGAACACACAATCATGTAACACAT
AAAAAACACGCTATAAAAAAACCATGTCTCTTATATAACAAAAAAACGCCGACAATTTAAAATGCCCTCT
TATTTTAAACAAAGAATAAACGACACCAGAAAAACAACTCAGCCTGACAGATTGCTTTCAAAACATCACC
TATAATTTATGGCATCATTTAAAACCTTAAGAAAATAAGGAAAGCAAATGAACTGGATAGTGATTGACAC
GGTAATCCAGCCAGCATGTGGCATATCATTTTCAGCCATATGGTGTAACATAAAATTAATATTATGGTAT
CAGTCAGATACCTTCCTCCCTCCGGGCTGTATTTTTACACCAACCCGGACAGGCGTAATACTCAATAATA
AAGAGCTCCCCGTCACCATTTACAACGTGGCCCCATTTAATAAAAACGTCTGGAATTTAATTAAAAACAG
CCAGGAGTGCCCTGCAAACACAGGAAATATCACAGATAAGTGTTTTAATCATCGCTGCATTCTGGAAATA
TGCCCATACGGGCGAAAATAACAGAATCATTCAGATAAAAAGCCCCTCCGGAGAGGGGCTGACACTGCGT
ATCTGTATCATCATGTACATGGTGCCGGGTGCCTCCCGGTGAGTTCAGTCCGGTGTCCCTGAACCCGCGT
ATCTCGCTCCAGGTTGTCGTCAGAGATGACACCTTATACACCAGTCGCCCCTCCGCACAGGGGGATTCAC
CATGCGAATTTTTTTAACGAATGCTCAGTCTGACAGGCAACTGTCAACTGACTGAATTGTGACACAGATT
ACACTTGTTACCCACATACCACGAATCAGGTTATGCCTCAGTCATTATTAAACTGCACTTCAGCAAATCC
GGAGCCTGATTCACAGGTACTGGATTTGATTGTGACAGTCATTCCTGTCAACTGAGCACTTTGCAGTAAC
GGTTGCAGATTCCAGCGACTGGTCCAGTACTCTTTTCCGGCCACTTTTACTGTGAATGTATCATTCTCAT
TATACTTGGAAAACTCAATTTTACCTTTAGCGCAATCCGCCGCCATTGCATTAACAGAAACTAATGCAAA
TAAAACCGCCATAAACATCTTCTTCATACTTAACTCCTTTATTTACCCGTTGTATATAAAAACTGTGACT
TTCTGTTCAGAAACGCTGCAGCTGTATTACTTTCCCATAATGTATTGTTTATTTTTATAACGGGCCTGTC
GCCAGTTATCTGACATTCTGGTTGACTATCTTCATTCACGGCGCGAACAGAACGCGCCCCCTGATGATGA
CAATTCAGTATAACGGCCACAGTGCCCAGTATCGCCGATATATTATTAAAGGATATTCTCCCCACTCTGA
CACCATCCTCTCCCCGATACTCCGGAAGCACATTGCTGATTCGCCCCCAGTTCAGAGTGAGGTCCACGTC
TCCCGGCGTCATCGTATACACAGGAGCAGTTTCAGACAGTGCCTGACGAAATTCTCTCTGTATCTGCCTG
AAGCGTAAGGCTTCTGCTGTGACAGTGACAAAACGCAGAACTGCTCTGGATGCATCTCTGGTCATTGTAT
TACCACTGAACTCCATTAACGCCAGATATGATGAAACCAGTGAGTGACGACTGATTTGCATTCCGGAACG
TTCCAGCGCTGCGACACGTTGCAGAGTGGTATAACTGCTGTCCGTTGTCATGGAAACCGTTGTCACACCG
GGCACTGATATATGTGTAAAATCTGAAAAACGGTAGAAAGTATTTGTTGCCGTATTAACGAACCCGGCCA
CATATAAATTATTTTGCTCAATAATCAGACGAAGATGGTCAAAACGCGCCTGATAGACATCAAGCCCTCG
TATATCCACAGCAAAATAACTGCCCGGTGGGGTGTGGTTAATAACAGACACCGATGTGGTCCCCTGAGAT
ATATGTTCAAGAGGGGTCGATATCTCTGTCCGTATACTATTTAACGAAGATACATAACTTTGTTGAGTCG
AAAAGTCTATCGTAAATTCCCGGGAATAGGATACCGAAGAAAAGCCCAGTAACAGGCACAGTACCCATTT
AAATAATATACACTTCATATACAAGGTGTTCCTTTTGGCTGAAGTAATCAGCACCAGACCCGGCGCAGAT
ATAAAAAAGGCCCGCAAAAGCGAGCCAGGGAAAATAAGTGTGGCGCGTTGTACTGGATTCGAACCAGTGA
CCGATTGCTTAGAAGGCAATTGCTCTGTCCTGCTGAGCTAACAACGCAGAATACCGATAATGGACCACCA
CCGGGGACTCGAACCTCGCGCAACCAGCTTCGAAGGCTGACGCTCTATCCCGATGAGCTAATGGCGGTTG
GTGGCCCTTGCTGGATTTGAACCAGCGACCTGGCGATTATGAGTCGCTCGCTCTCACCACTGAGCTAAAG
GGCC

>partial_contig_end gb|AAMEYR010000123.1| Salmonella enterica subsp. enterica serovar Senftenberg strain 2017K-0250 SAMN06761671-rid5489163.denovo.123, whole genome shotgun sequence
GAAAACTCAATTTTACCTTTAGCACAATCCGCCGCCATTGCATTAACAGAAGCTAATGCAAATAAAACCG
CCATAAACATCTTCTTCATGCTTAACTCCTTTATTTACCCGTTGTATATAAAAACTGTGACTTTCTGTTC
AGAAACGCTGCAGCTGTATTACTTTCCCATAATGTATTGTTTATTTTTATAACAGGCCTGTCGCCAGTTA
TCTGACATTCTGGTTGACTCTCTTCATTCACGGCGCGAACAGAACGCGCCCCCTGATGATGGCAATTCAG
TATAACGGCCACAGTCCCCAGTATCGCTGATATATTATTAAAGGATATTCTCCCCACTCTGACACCATCC
TCTCCCCGATACTCCGGAAGCACATTGCTGATTCGCCCCCAGTTCAGAGTGAGGTCCACGTCTCCCGGCG
TCATCGTATACACAGGAGCAGTTTCAGACAGTGCCTGACGAAATTCTCTCTGTATCTGCCTGAAGCGTAA
GGCTTCTGCTGTGACAGTGACAAAACGCAGAACTGCTCTGGATGCATCTCTGGTCATTGTATTACCACTG
AACTCCATTAACGCCAGATATGATGAAACCAGTGAGTGACGACTGATTTGCATTCCGGAACGTTCCAGCG
CTGCGACACGTTGCAGAGTGGTATAACTGCT

>partial  gb|AATDDC010000166.1| Escherichia coli strain 2014C-3076 SAMN04502368-rid9102293.guided.129, whole genome shotgun sequence
ACAAATGGTAGTGTCCGCTATTGCCAGTACACCTCAGAGGGCTTGATGTCTATCAGGCGCGTTTTGACCA
TCTTCGTCTGATTATTGAGCAAAATAATTTATATGTGGCCGGGTTCGTTAATACGGCAACAAATACTTTC
TACCGTTTTTCAGATTTTACACATATATCAGTGCCCGGTGTGACAACGGTTTCCATGACAACGGACAGCA
GTTATACCACTCTGCAACGTGTCGCAGCGCTGGAACGTTCCGGAATGCAAATCAGTCGTCACTCACTGGT
TTCATCATATCTGGCGTTAATGGAGTTCAGTGGTAATACAATGACCAGAGATGCATCCAGAGCAGTTCTG
CGTTTTGTCACTGTCACAGCAGAAGCCTTACGCTTCAGGCAGATACAGAGAGAATTTCGTCAGGCACTGT
CTGAAACTGCTCCTGTGTATACGATGACGCCGGGAGACGTGGACCTCACTCTGAACTGGGGGCGAATCAG
CAATGTGCTTCCGGAGTATCGGGGAGAGGATGGTGTCAGAGTGGGGAGAATATCCTTTAATAATATATCA
GCGATACTGGGGACTGTGGCCGTTATACTGAATTGCCATCATCAGGGGGCGCGTTCTGTTCGCGCCGTGA
ATGAAGAGAGTCAACCAGAATGTCAGATAACTGGCGACAGGCCCGTTATAAAAATAAACAATACATTATG
GGAAAGTAATACAGCTGCAGCGTTTCTGAACAGAAAGTCACAGTTTTTATATACAACGGGTAAATAAAGG
AGTTAAGCATGAAGAAGATGTTTATGGCGGTTTTATTTGCATTAGCTTCTGTTAATGCAATGGCGGCGGA
TTGTGCTAAAGGTAAAATTGAGTTTTCCAAGTATAATGAGGATGACACATTTACAGTGAAGGTTGACGGG
AAAGAATACTGGACCAGTCGCTGGAATCTGCAACCGTTACTGCAAAGTGCTCAGTTGACAGGAATGACTG
TCACAATCAAATCCAGTACCTGTGAATCAGGCTCCGGATTTGCTGAAGTGCAGTTTAATAATGACTGAGG
CATAACCTGATTCGTGGTATGTGGGTAACAAGTGTAATCTGTGTCACAATTCAGTCAGTTGACAGTTGCC
TGTCAGACTGAGCATTTGTTAAAAAAATTTCGCATGGTGAATCCCCCTGTGTGGAGGGGCGACTGGTGAA
AAATCCTTGCTTGTGATTCATTATCGACACGGGTTCGGTGGTACCAAGCCGAACTCACCGGGAGGCACCC
GGCACCATGCAGTATACAGAGATTAGGCATATACCAAGGCCTCTCATAGCAGGGGCCTTTTTACATGTAA
AAAAAGCCCGAGTGGGTTCGGGCAATTGCATGAGATACTCGTTTTAATAATCGAAAGCATTTTAACCAGG
ATTCATAAGGCTGCGCAACTGCGCGGCCTTTTTCGTATTTCGGGCTGTAGTCTTCCTTCTGCCATTGTCC
TGTAACTTCCGGACTTCAGCCCGCT

