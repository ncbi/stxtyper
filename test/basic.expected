#target_contig	stx_type	operon	identity	target_start	target_stop	target_strand	A_reference	A_reference_subtype	A_identity	A_coverage	B_reference	B_reference_subtype	B_identity	B_coverage
partial	stx2	PARTIAL	99.41	27	1048	+	AAA16362.1	stxA2c	99.19	77.19	AAS07607.1	stxB2a	100.00	100.00
partial_contig_end	stx2	PARTIAL_CONTIG_END	100.00	3	661	-	AAA16362.1	stxA2c	100.00	58.44	AAM70046.1	stxB2a	100.00	32.22
stx1a	stx1a	COMPLETE	100.00	218	1444	+	AAA98347.1	stxA1a	100.00	100.00	AAA71894.1	stxB1a	100.00	100.00
stx2_fs	stx2	FRAMESHIFT	99.15	2165	3232	+	AAG01033.1	stxA2c	98.87	82.19	AAA16363.1	stxB2c	100.00	100.00
stx2_novel	stx2	COMPLETE_NOVEL	99.76	216	1456	+	AAA19623.1	stxA2	99.69	100.00	AAA16363.1	stxB2c	100.00	100.00
stx2_stop	stx2	INTERNAL_STOP	NA	694	1635	+	WAK53220.1	stxA2n	100.00	100.00	NA	NA	NA	NA
stx2c	stx2c	COMPLETE	100.00	1298	2538	-	AAS07596.1	stxA2	100.00	100.00	AAA16363.1	stxB2c	100.00	100.00
